core/src/main/java/com/scalar/db/storage/cassandra/SelectStatementHandler.java

```diff
@@ -294,8 +289,7 @@ private Ordering getOrdering(Scan.Ordering ordering) {
       case DESC:
         return QueryBuilder.desc(quoteIfNecessary(ordering.getColumnName()));
       default:
-        logger.warn("Unsupported ordering specified. Using Order.ASC");
-        return QueryBuilder.asc(quoteIfNecessary(ordering.getColumnName()));
+        throw new AssertionError("Unsupported ordering is specified");
```

Comment:
Minor thing, but It's better to add the value to the message as follows:
```suggestion
        throw new AssertionError("Unsupported ordering is specified: " + ordering.getOrder());
```